cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -635,11 +635,8 @@ pub mod bridging {
 		use super::*;
 
 		parameter_types! {
-			/// User fee for ERC20 token transfer back to Ethereum.
-			/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/400 and fee_per_gas 20 GWEI = 2200698000000 + *25%)
-			/// Needs to be more than fee calculated from DefaultFeeConfig FeeConfigRecord in snowbridge:parachain/pallets/outbound-queue/src/lib.rs
-			/// Polkadot uses 10 decimals, Kusama and Rococo 12 decimals.
-			pub const DefaultBridgeHubEthereumBaseFee: Balance = 2_750_872_500_000;
+			/// User fee for delivery cost on bridge hub. Leave some buffer here for avoid spamming
+			pub const DefaultBridgeHubEthereumBaseFee: Balance = 1_000_000_000;
```

Comment:
Should at least cover the execution cost of the fee on BH(the local portion).

Reply:
Ah, we should not bake this constant into AH. BH needs to estimate it on-chain, and then our off-chain UX needs to query it in order to estimate the total fees required

Reply:
The XCM fee estimation APIs will include the local fee, so we should be fine in that regard.

Reply:
The fee config here will also be used to [BuyExecution](https://github.com/Snowfork/polkadot-sdk/blob/f0abb7ea9b252db1fd2bbaa05fc6569e7c96e009/polkadot/xcm/xcm-builder/src/universal_exports.rs#L307-L308) which can't be zero.

It should at least cover the xcm execution cost but I would suggest to leave some buffer to also include the local portion(i.e. the submit cost) here to ensure our `OutboundQueue` not spammed.